From c4b706634595c9c7f46b0b74de516e1a59220d6d Mon Sep 17 00:00:00 2001
From: zhangping <15774560747@163.com>
Date: Tue, 28 Jul 2020 18:09:18 +0800
Subject: [PATCH] format-patch test

---
 a.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/a.txt b/a.txt
index 66b42f8..0ae3962 100644
--- a/a.txt
+++ b/a.txt
@@ -1,2 +1,3 @@
 This is the file a.
 fix !!!!!
+the other test about format-patch!!!
-- 
1.8.3.1

